roles/clevis/README.rst

```diff
@@ -0,0 +1,37 @@
+An ansible role to install clevis library for network bound disk encryption.
+
+**Role Variables**
+
+.. zuul:rolevar:: clevis_packages
+   :default: []
+
+   the clevis required installation packages
+
+.. zuul:rolevar:: clevis_tang_server
+   :default: 127.0.0.1
+
+   the tang server ip required create for tang clevis chain creation
+
+.. zuul:rolevar:: clevis_tang_server_protocol
+   :default: 80
+
+   the tang server protocol port required create for tang clevis
+   chain creation
+
+.. zuul:rolevar:: clevis_tang_server_url
+   :default: 80
+
+   the tang server url required create for tang clevis chain creation
+
+.. zuul:rolevar:: clevis_root_disk
+   :default: /dev/sda1
+
+   the crypt disk is required to create tang clevis chain creation
+
+.. zuul:rolevar:: clevis_initial_luks_password
+   :default: []
+
+   the tang server port required create for tang clevis chain creation
+
+   The Path where the temporary passfile with the initial_luks_password is stored,
```

Comment:
Seems this is a leftover from a previous version

Reply:
done